Kitodo/src/main/java/org/kitodo/production/services/data/MassImportService.java

```diff
@@ -99,10 +107,17 @@ public List<CsvRecord> parseLines(List<String> lines, String separator) throws I
                      .withSkipLines(0)
                      .withCSVParser(parser)
                      .build()) {
+            int targetNumberOfColumns = -1;
             for (String[] entries : csvReader.readAll()) {
                 if (isSingleEmptyEntry(entries)) {
```

Comment:
This is already in the old code, but i noticed that lines like 
```
;;;;;;;;;;;;;;;;;;
```
get included in the table as empty lines, maybe we should adapt the check to exclude all empty lines:

```java
 for (String[] entries : csvReader.readAll()) {
                if (isLineCompletelyEmpty(entries)) {
                ..
                }
}

private boolean isLineCompletelyEmpty(String[] entries) {
    return entries.length == 0 || 
           Arrays.stream(entries).allMatch(StringUtils::isBlank);
}
```

Reply:
You are right. I added the additional check to discard completely empty lines.